src/TimeSync.cpp

```diff
@@ -0,0 +1,91 @@
+#include "TimeSync.hpp"
+
+namespace carma_streets_time_sync {
+    TimeSync::TimeSync(const std::string &ip , unsigned int port ) : initialized(false), _ip(ip), _port(port)  {}
+ 
+    void TimeSync::start() {
+        if (!initialized) {
+            // Intialize carma clock
+            std::string sim_mode_string = getSystemConfig("SIMULATION_MODE","TRUE");
```

Comment:
Can we add a note in the readme.md or include a sample.env file to indicator some environment variable dependencies needed for this lib?